google-chart-loader.html

```diff
@@ -0,0 +1,206 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../promise-polyfill/promise-polyfill-lite.html">
+<link rel="import" href="charts-loader.html">
+
+<script>
+(function() {
+  "use strict";
+
+  // Most charts use this package.
+  var DEFACTO_CHART_PACKAGE = 'corechart';
+
+  // A collection of chart types associated with a function that returns a
+  // Promise for the chart type's constructor.
+  var CHART_CONSTRUCTORS = {
+    'area': {
+      ctor: 'AreaChart',
+    },
+    'bar': {
+      ctor: 'BarChart',
+    },
+    'md-bar': {
+      ctor: 'Bar',
+      pkg: 'bar',
+    },
+    'bubble': {
+      ctor: 'BubbleChart',
+    },
+    'candlestick': {
+      ctor: 'CandlestickChart',
+    },
+    'column': {
+      ctor: 'ColumnChart',
+    },
+    'combo': {
+      ctor: 'ComboChart',
+    },
+    'geo': {
+      ctor: 'GeoChart',
+    },
+    'histogram': {
+      ctor: 'Histogram',
+    },
+    'line': {
+      ctor: 'LineChart',
+    },
+    'pie': {
+      ctor: 'PieChart',
+    },
+    'scatter': {
+      ctor: 'ScatterChart',
+    },
+    'stepped-area': {
+      ctor: 'SteppedAreaChart',
+    },
+    'table': {
+      ctor: 'Table',
+      pkg: 'table',
+    },
+    'timeline': {
+      ctor: 'Timeline',
+      pkg: 'timeline',
+    },
+    'gauge': {
+      ctor: 'Gauge',
+      pkg: 'gauge',
+    },
+    'treemap': {
+      ctor: 'TreeMap',
+      pkg: 'treemap',
+    },
+  };
+
+  function namespaceForType(type) {
+    return google[type.startsWith('md-') ? 'charts' : 'visualization'];
+  }
+
+  var packagesToLoad = {};
+  var promises = {};
+  var resolves = {};
+
+  Polymer({
+    is: 'google-chart-loader',
+    properties: {
+      /**
+       * Adds packages to the list of packages to load.
+       *
+       * This is an array consisting of any Google Visualization package names.
+       *
+       */
+      prime: {
```

Comment:
Agreed. I was trying to relay that this would be for 'priming' packages from the beginning, but I wasn't really sold on it.